### PR TITLE
Added bunlle analyzer; removed unused react from package json;

### PR DESCRIPTION
**Pull request by rfreitas**
# Problem

1.  We needed a better inspection tool of our dependencies
2. Some dependencies in package.json might not really be used
3. Motivated by https://github.com/onfido/onfido-sdk-ui/issues/317

# Solution
1. Removed React from package.json
2. Added bundle analyser for both dist and npm distributions

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [n/a] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [n/a] Have new automated tests been implemented?
- [n/a] Have new manual tests been written down?
- [n/a] Have tests passed locally?
- [n/a] Have any new strings been translated?

### CODE REVIEW

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-319-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-319-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://319-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Bundles: https://319-pr-onfido-sdk-ui-onfido.surge.sh/reports/bundle_dist_size.html

File: webpack.config.babel.js [hunk 1]
```diff
@@ -128,7 +131,12 @@ const formatDefineHash = defineHash =>
     value => JSON.stringify(value)
   )
 
-const basePlugins = [
+const basePlugins = (bundle_name) => ([
+  new BundleAnalyzerPlugin({
+    analyzerMode: 'static',
```
Comment (stefaniacardenas) [inline webpack.config.babel.js]: What's the `analyzerMode` for?

Comment (rfreitas) [reply webpack.config.babel.js]: https://www.npmjs.com/package/webpack-bundle-analyzer
It creates a report file

**Pull request merged**